### PR TITLE
Fix jenkinsfile

### PR DESCRIPTION
**Pull request by ethernetdan**
New from #269

### CODE REVIEW

**Comment by Quentin-M** [comment]
Answered in DM. This is due to the Kubernetes cluster being named differently in that Jenkins environment.

**Comment by ethernetdan** [comment]
@Quentin-M what does it need to be named? or should I just not set it?

**Comment by ethernetdan** [comment]
FYI, currently set [here](https://github.com/coreos/tectonic-installer/blob/fe15418/Jenkinsfile#L120)

**Comment by Quentin-M** [comment]
It's not that. Jenkins is trying to find the configured cloud provider to launch Kubernetes pod to build the Tectonic Installer. In this Jenkins env, there is no `kubernetes` provider, but there is another one, named differently (by the actual cluster name). Either rename that provider to `kubernetes`, or either configure the Jenkinsfile to use the right name. It is not related to TF or to the variable you pointed out.

File: Jenkinsfile [hunk 1]
```diff
@@ -19,88 +19,34 @@ pipeline {
...
-              containerTemplate(
-                name: 'webapp-agent',
-                image: 'quay.io/coreos/tectonic-builder:v1.4',
-                ttyEnabled: true,
-                command: 'cat',
-              )
-            ],
-            volumes: []
-          ) {
-            node('webapp-pod') {
```
Comment (dghubble) [inline Jenkinsfile]: Is it intentional you've stopped running these containers on Kubernetes?

Comment (ethernetdan) [inline Jenkinsfile]: For now until we can get the executors running correctly

**Comment by ethernetdan** [comment]
Looks like this works!

@sym3tri @Quentin-M @dghubble good to merge?

**Comment by ethernetdan** [comment]
Merging to get working validation running, will revisit with @dghubble comments from #207

**Pull request merged**